codecov.yml

```diff
@@ -37,19 +37,15 @@ flag_management:
     - name: api
       paths:
         - flask/
-      statuses:
-        - type: project
-          target: auto
-          threshold: 1%
     - name: frontend
       paths:
         - react/
-      statuses:
-        - type: project
-          target: auto
-          threshold: 1%
   default_rules:
     carryforward: true
+    statuses:
```

Comment:
Let's move this out of this PR -- make a separate PR and merge to `main`, so this change will go away. Want to only show 'real code changes' in this PR.